File: 1475. Final Prices With a Special Discount in a Shop.py

class Solution:
    def finalPrices(self, prices: List[int]) -> List[int]:
        for i in range(len(prices)):
            for j in range(i+1,len(prices)):
                if prices[i]>=prices[j]:
                    prices[i] -=prices[j]
                    break
        return prices

File: 1475. Final Prices With a Special Discount in a Shop.go
func finalPrices(prices []int) []int {
    arrLen := len(prices)
	for i:=0; i<arrLen; i++{
		for j:=i+1; j<arrLen; j++{
			if prices[i]>= prices[j]{
				prices[i] = prices[i] - prices[j]
				break
			}
		}
	} 
	return prices
}
